data/src/main/java/com/android254/data/network/models/SessionDataModel.kt

```diff
@@ -0,0 +1,22 @@
+/*
+ * Copyright 2022 DroidconKE
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.android254.data.network.models
+
+/* TODO */
```

Comment:
Why do we have this?

Reply:
The model isn't complete. I only added what I needed for the task I was working on since the model looks to me to be outside the scope of this PR and probably part of another task.
I thought it would be important to make it explicit that "this isn't the final thing"; that's why I marked it with "TODO".

What do you think? Do I still remove it?

Reply:
@johnGachihi Please remove it.